src/rules/double-slash-comment-whitespace-inside/README.md

```diff
@@ -0,0 +1,60 @@
+# double-slash-comment-whitespace-inside
+
+Require or disallow whitespace after the `//` in `//`-comments
+
+```scss
+a {
+  width: 10px; // inline-comment
+/*               ↑
+ * Such whitespace */
+```
+
+This rule only works with SCSS-like [single-line comments](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments) and ignores CSS comments (`/* */`).
+
+Any number of slases are allowed at the beginning of the comment. So `/// comment` is treated the same way as `// comment`.
```

Comment:
I think it's not that relevant in the other rules. Here one might think that `/// comment` is a warning with "always" because there is a 3rd slash after the `//`. So the extra explanation is given. For checking newlines before a comment and if a comment is a newline it is irrelevant what exactly goes after the `//` marker.